.env.template

```diff
@@ -9,4 +9,17 @@ DEVISE_SECRET_KEY=<LOCKBOX_MASTER_KEY_REPLACE_ME>
 # Used for Github integration
 # See https://docs.autolabproject.com/installation/github_integration
 GITHUB_CLIENT_ID=
-GITHUB_CLIENT_SECRET=
\ No newline at end of file
+GITHUB_CLIENT_SECRET=
+
+# Used to connect Tango with autolab
+# Defaults for development are
+# Ensure variables are consistent with with your Tango install's `config.py` settings
```

Comment:
line 15: either delete, or comment the default values below

line 16 typo: "with with"

Reply:
fixed